addons/project_timesheet_holidays/views/project_task_views.xml

```diff
@@ -7,7 +7,7 @@
         <field name="inherit_id" ref="hr_timesheet.view_task_form2_inherited"/>
         <field name="arch" type="xml">
             <xpath expr="/form" position="inside">
-                <field name="is_timeoff_task" invisible="1"/>
+                <field name="is_timeoff_task" invisible="1" groups="hr_timesheet.group_hr_timesheet_user"/>
             </xpath>
             <xpath expr="//field[@name='timesheet_ids']" position="attributes">
                 <attribute name="readonly">not analytic_account_active or is_timeoff_task</attribute>
```

Comment:
Shouldn't we also add the group to `timesheet_ids` ? As it needs to read `is_timeoff_task`. I'm not sure tho :thinking:

We may also want to add the group in the Python field declaration of `is_timeoff_task` instead but I'm not sure it's doable in stable.